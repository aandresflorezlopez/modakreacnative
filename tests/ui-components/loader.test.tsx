import { render, screen } from '@testing-library/react-native';
import Loader from '../../src/ui-components/loader';

test('renders loader component correctly', async () => {
  render(<Loader />);

  expect(screen.toJSON()).toMatchSnapshot();
});
