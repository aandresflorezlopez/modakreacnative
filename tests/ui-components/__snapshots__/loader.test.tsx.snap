// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loader component correctly 1`] = `
<View
  collapsable={false}
  style={
    {
      "alignSelf": "center",
      "flex": 1,
      "justifyContent": "center",
      "opacity": 0,
    }
  }
>
  <Image
    source={
      {
        "testUri": "../../../assets/waiting.png",
      }
    }
    style={
      [
        {
          "height": 50,
          "paddingtop": "10em",
          "pagging": "20em",
          "width": 50,
        },
      ]
    }
  />
</View>
`;
